src/components/Header.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './styles/Header.css';
+
+export default function Header() {
+  return (
+    <div className={styles.Header}>
```

Comment:
```suggestion
    <nav className={styles.Header}>
```
Nav tag may be more appropriate here.